lib/auth0/api/v2/connections.rb

```diff
@@ -29,14 +51,37 @@ def connection(connection_id, fields: nil, include_fields: true)
           get(path, request_params)
         end
 
+        # Deletes a connection and all its users.
+        # @see https://auth0.com/docs/api/v2#!/Connections/delete_connections_by_id
+        # @param connection_id [string] The id of the connection to delete
         def delete_connection(connection_id)
-          fail Auth0::MissingConnectionId, 'you must specify a connection id' if connection_id.to_s.empty?
+          fail Auth0::InvalidParameter, 'Must supply a valid connection id' if connection_id.to_s.empty?
           path = "/api/v2/connections/#{connection_id}"
           delete(path)
         end
 
+        # Deletes a specified connection user by its email (currently only database connections are supported and you
+        # cannot delete all users from specific connection).
+        # @see https://auth0.com/docs/api/v2#!/Connections/delete_users
+        # @param connection_id [string] The id of the connection
+        # @param user_email [string] The email of the user to delete
+        #
+        # @return  [json] Returns the updated connection.
+        def delete_connection_user(connection_id, user_email)
+          fail Auth0::InvalidParameter, 'Must supply a valid connection id' if connection_id.to_s.empty?
+          fail Auth0::InvalidParameter, 'Must supply a valid user email' if user_email.to_s.empty?
+          path = "/api/v2/connections/#{connection_id}/users?email=#{user_email}"
+          delete(path)
+        end
+
+        # Updates a connection. Updates the fields specified in the body parameter.
+        # @see https://auth0.com/docs/api/v2#!/Connections/patch_connections_by_id
+        # @param connection_id [string] The id of the connection to delete
+        # @param body [hash] The Hash options used to update the conecctions's properties.
+        #
+        # @return  [json] Returns the updated connection.
         def update_connection(connection_id, body)
-          fail Auth0::MissingConnectionId, 'you must specify a connection id' if connection_id.to_s.empty?
+          fail Auth0::InvalidParameter, 'Must supply a valid connection id' if connection_id.to_s.empty?
```

Comment:
Most of the errors are 'Must....;, so we'll keep this one, but we should change it in clients.rb